packages/cli/test/fixtures/unit/vercel-env-pull-delta-quotes/.env

```diff
@@ -0,0 +1,2 @@
+NEW_VAR=testvalue
```

Comment:
What about a test when the env var contains quotes?

Reply:
Added test 👍